src/elm/Page/FormPlayground.elm

```diff
@@ -0,0 +1,292 @@
+module Page.FormPlayground exposing (Model, Msg, init, msgToString, update, view)
+
+-- TODO - Remove this before merging PR
+
+import Eos
+import Form exposing (Form)
+import Form.Text
+import Html exposing (Html, div, p, strong, text)
+import Html.Attributes exposing (autocomplete, class, rows)
+import Icons
+import Mask
+import Session.LoggedIn as LoggedIn
+import Session.Shared as Shared
+import UpdateResult as UR
+import View.Form
+
+
+
+-- INIT
+
+
+init : LoggedIn.Model -> ( Model, Cmd Msg )
+init _ =
+    ( { userFormModel =
+            Form.init
+                { name = ""
+                , middleName = ""
+                , phoneNumber = ""
+                , age = ""
+                , bio = "This is an example bio!"
+                , balance = "0"
+                , disabledField = "This field is disabled"
+                }
+      , user = Nothing
+      }
+    , Cmd.none
+    )
+
+
+type alias Model =
+    { userFormModel : Form.Model DirtyUser
+    , user : Maybe User
+    }
+
+
+
+-- UPDATE
+
+
+type Msg
+    = GotUserFormMsg (Form.Msg DirtyUser User)
+    | SubmittedUser User
+
+
+type alias UpdateResult =
+    UR.UpdateResult Model Msg (LoggedIn.External Msg)
+
+
+update : Msg -> Model -> LoggedIn.Model -> UpdateResult
+update msg model _ =
+    case msg of
+        GotUserFormMsg subMsg ->
+            Form.update subMsg model.userFormModel
+                |> UR.fromChild
+                    (\userFormModel -> { model | userFormModel = userFormModel })
+                    GotUserFormMsg
+                    SubmittedUser
+                    model
```

Comment:
@lucca65 this is what I was talking about having a component using `UpdateResult`. In this case, whenever a valid `User` is submitted, the `SubmittedUser` message is fired, and then the parent (this page) can do whatever it wants with the user